bigtable/bigtable.go

```diff
@@ -437,6 +440,9 @@ func (t *Table) readRows(ctx context.Context, arg RowSet, f func(Row) bool, mt *
 		for {
 			proto.Reset(res)
 			err := stream.RecvMsg(res)
+			if err == nil || err == io.EOF {
+				mt.currOp.setFirstRespTime(time.Now())
```

Comment:
This is inside of the for loop, would it get updated on every response? How difficult would it be to add a test: emulator sends back 2 responses, sleep before sending the second response, client verifies that the recorded value is <= operation time - sleep before the second response?